ads/eas.md

```diff
@@ -0,0 +1,55 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Cxense Display ad server (formerly Emediate Ad)
+
+## Supported parameters in the amp-ad tag
+
```

Comment:
Ok, changed title to just "CxenseDisplay" instead, I hope that was what you where referring to (otherwise let me know what's wrong)